funnel/templates/otpform.html.jinja2

```diff
@@ -2,7 +2,7 @@
 {% from "forms.html.jinja2" import renderform, renderform_inner %}
 
 {% block form %}
-  <form data-parsley-validate="true" id="{{ ref_id }}" method="post" {%- if action %} action="{{ action }}" {%- endif %}{%- if multipart %} enctype="multipart/form-data" {%- endif %} accept-charset="UTF-8" class="mui-form mui-form--margins hg-form">
+  <form data-parsley-validate="true" id="{{ ref_id }}" method="post" hx-post="{{ url_for('login', next=request.path) }}%23register-modal" {%- if action %} action="{{ url_for('login', next=request.path) }}%23register-modal" {%- endif %}{%- if multipart %} enctype="multipart/form-data" {%- endif %} accept-charset="UTF-8" class="mui-form mui-form--margins hg-form" hx-target="#loginformwrapper">
```

Comment:
OTP form is also used for `sudo`, where it does not redirect to login. This one should not hardcode login URL.